hyperspy/_signals/complex_signal.py

```diff
@@ -260,6 +261,59 @@ def angle(self, deg=False):
         return super().angle(angle, deg=deg)
     angle.__doc__ = ComplexSignal_mixin.angle.__doc__
 
+    def argand_diagram(self, size=[256, 256], display_range=None):
```

Comment:
Instead of `display_range`, I would suggest to use `range`. Using _display_ implies that this argument is intented to be used to adjust the display of something, while here it actually defines how the output signal is calculated. And this would make it consistent with the corresponding one of `np.histogram2d`.

Reply:
Agreed in principle, but I don't like using `range` since it shadows python built-in function. Do you think it is fine?

Reply:
As far as I know, this is not a problem and https://docs.scipy.org/doc/numpy/reference/generated/numpy.histogram.html uses `range` as argument, in my opinion, it would be logical to follow a similar convention here.

Reply:
I'm tempted to suggest "window_limits" or "limits" instead, since `range` hints at a different type of iterable.